tests/test_datastructures.py

```diff
@@ -210,6 +213,17 @@ def test_queryparams():
     assert QueryParams(q) == q
 
 
+@pytest.mark.asyncio
+async def test_upload_file():
+    UploadFile.spool_max_size = 1024
```

Comment:
We're changing this class attribute across all the test cases. Probably doesn't really matter, but we could alternately do something like...

```python
TestUploadFile(UploadFile):
    spool_max_size = 1024
```

And use that?
Also perhaps we should assert that the `.read` is returning the expected data?

Reply:
I agree. I will modify the test code here.